build.gradle

```diff
@@ -179,6 +179,7 @@ dependencies {
 
     compile project(":common")
     compile project(":cassandra40")
+    compile 'org.apache.cassandra:cassandra-all:4.0-beta2'
```

Comment:
If you take a look at the last commit I added, I spent a lot of time trying to decouple the sidecar from using a specific version of Cassandra.  Each version we decide to support can (and will) have an adapter, allowing us to maintain a single sidecar project that can work with different versions of Cassandra each of which has different implementations.  There's no assurance that C* 5.0 will have the same CDC implementation as the 4.0 version.  Could you please move the version specific logic into the cassandra40 subproject?

In addition, we may want to have the user point to their cassandra lib directory as well in order to not ship every version of C* with the sidecar.  That will give us the flexibility for folks to use their own builds (private or public) as well as ship a smaller artifact.  Since everyone *has* to run Cassandra I think this is a fair ask.  Using a `compileOnly` dependency would allow us to test against each version of C* without shipping the jars.

Reply:
@rustyrazorblade +1 for both suggestions. I was thinking of punting this to a future commit, but I see the work you've done at `a4805a910904019698ae373ac33f88855cf67f3d`. Let me refactor this code to address both points.

Reply:
@rustyrazorblade +1